projects/gateway2/translator/testutils/outputs/tcp-routing/invalid-backend.yaml

```diff
@@ -5,8 +5,6 @@ listeners:
   bindPort: 8080
   aggregateListener:
     httpResources: {}
-    tcpListeners:
```

Comment:
Given that we're deleting this, can we we add a comment in the yaml to clarify why this resource contains an invalid backend? In fact, this looks identical to the file with a missing backend

Reply:
Resolved by commit https://github.com/solo-io/gloo/commit/281ace000966aa7f15d960ab73d60efc4fa1c189 by adding comments to testdata.